kong/tools/aws_stream.lua

```diff
@@ -0,0 +1,186 @@
+--- Stream class.
```

Comment:
need a unit test for this module

Reply:
It's all tested in https://github.com/Kong/kong/blob/fa52c16586774a6629993e80b46cc5f2d27a18e7/spec/03-plugins/38-ai-proxy/01-unit_spec.lua#L677

Is that enough?

I didn't add specific units for this, because they're already in lua-resty-aws (but we can't wait for that package release...)